packages/local_auth/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.5.2
```

Comment:
Ideally this version would actually be `0.5.1+1` since it's a bugfix. This is technically incrementing the minor digit. Reason is that pub has inconsistent semantic versioning policies depending on how mature the package is. For packages that are < `1.0.0` it uses `0.major.minor+patch` for semver, and then for anything at or above 1 it uses the standard `major.minor.patch`. Not a big deal though.

https://dart.dev/tools/pub/versioning#semantic-versions